OpenRA.sln

```diff
@@ -1,6 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 2013
+VisualStudioVersion = 12.0.31101.0
+MinimumVisualStudioVersion = 10.0.40219.1

```

Comment:
I'm being extremely lazy in this PR and including all the stuff VS liked to change arbitrarily.

I could happily leave it out, on the other hand it would be nice to eliminate some of the stuff the IDE changes all the time.

I don't know what effect this setting has, but given that VS is free these days and people can get 2013 (and soon 2015) easily enough, it probably doesn't matter?